Formula/dbus.rb

```diff
@@ -22,10 +22,18 @@ class Dbus < Formula
     url "https://anongit.freedesktop.org/git/dbus/dbus.git"
 
     depends_on "autoconf" => :build
+    depends_on "autoconf-archive" => :build
     depends_on "automake" => :build
     depends_on "libtool" => :build
   end
 
+  depends_on "xmlto" => :build
+
+  # Docbook is a dependency of xmlto, but the XML_CATALOG_FILES env-set in
+  # install() uses files created by docbook, so best to be explicit in case
+  # the install process for docbook ever stops creating those files.
```

Comment:
Let's drop this entire comment and the explicit docbook dependency. ascii, deheader, pgbouncer, reposurgeon, and shadowsocks-libev all have the same situation and none declares a docbook dependency.